src/core/Details/Details.md

```diff
@@ -0,0 +1,70 @@
+Examples:
```

Comment:
Every other component has a short description text at the top of the documentation page. This should probably too

Reply:
Good point. Added one and some usage instructions to boot.